### DISCUSSION TITLE
About model convert

### DISCUSSION THREAD

**Discussion by Youskrpig** [open-edge-platform/anomalib#118] [Q&A]
Is there any supplement about .ckpt converted to .onnx, .bin or .xml?

**Comment by samet-akcay** [comment]
Hi @Youskrpig, can you please elaborate?

**Comment by Youskrpig** [comment]
> Hi @Youskrpig, can you please elaborate?

Ummm. when i finishing training and testing, I get .ckpt of model. How can i  convert it to other format such as .onnx,.bin or .xml? Is there any instruction?

**Comment by samet-akcay** [comment]
Ah I see. There are some instructions in the [Inference](https://github.com/openvinotoolkit/anomalib#inference) section in the README.md file. Basically, in the `config.yaml` file, you need to set the compression `True`

```
optimization:
  compression:
    apply: true
```

For instance, say you use Padim, set apply compression to True in `anomalib/models/padim/config.yaml`. It should generate the models.

Let us know how it goes.

**Reply by Youskrpig** [reply]
appreciate!
